### ISSUE TITLE
Display Name is overwritten by First Name / Last Name changes.

### ISSUE THREAD

**Issue by rNoz** [nextcloud/contacts#361]
### Steps to reproduce
1. Create a new contact.
2. Fill the Display name (top menu, above the Organization and Title)
3. Add a new field (detailed name) and fill First Name and Last Name.

### Expected behaviour
Display name shouldn't be overwritten with the First Name + Last Name. It should be written in Display Name just if this field is empty.

### Actual behaviour
Display name is overwritten with First Name + Last Name.

**Comment by irgendwie** [comment]
Hello! Thanks for reporting this issue - unfortunately the actual behavior isn't always the desired behavior. Think of Mistakes a user makes while inputting a detailed name or a changing title, etc. - this would lead to two places the user needs to adapt changes afterwards.
We will try to improve name handling for version 2.1.0.

ref #307

**Comment by rNoz** [comment]
Thanks for your response. That is true, but it is quite uncomfortable to have a Display Name, and just adding something to the Detailed name (suffix, last name, second last name, etc) removing and overwritting the Display Name.

If you ask me I would do: on every field change in Detailed Name you can check if Display Name is empty. if it is, do the actual behavior. if it is not, just leave it without modifying it.

Thanks again.

**Comment by irgendwie** [comment]
Of course! The latter is already implemented. The update on detail name change was a feature implemented a longer time ago - we'll try to come up with a better solution, suitable for everyone or most of the users.

**Comment by rNoz** [comment]
Thanks. I updated what I would do because makes more sense.
